cmd/evm/internal/t8ntool/execution.go

```diff
@@ -272,6 +278,10 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		GasUsed:     (math.HexOrDecimal64)(gasUsed),
 		BaseFee:     (*math.HexOrDecimal256)(vmContext.BaseFee),
 	}
+	if pre.Env.Withdrawals != nil {
```

Comment:
Yeah I had it this way so that it was possible to generate a post-shanghai block with no wx root, but I guess that doesn't make much sense + we already error out if `currentbaseFee` is not set post-london. I will update.